lib/outbrain/request.rb

```diff
@@ -17,6 +17,16 @@ def self.all(resource, options={})
       where(resource, {}, options)
     end
 
+    #token retrieval
+    def self.get(resource, attributes)
+      user_name = attributes["user_name"] || attributes[:user_name]
+      user_password = attributes["user_password"] || attributes[:user_password]
```

Comment:
I'm not sure that this project is using Rails. If it isn't then you can't use that 😞